File: seungyeop/22_11_18_01.py
def solution(my_string):
    vowels = ["a", "e", "i", "o" , "u"] # string으로 "aeiou"로 적어도 문제 없었다.
    for vow in vowels:
        my_string = my_string.replace(vow, "")
            
    return my_string

File: seungbeom/22_11_18_01.java
// https://school.programmers.co.kr/learn/courses/30/lessons/120849

import java.util.HashSet;

// 풀이
class Solution {
    public String solution(String my_string) {
        StringBuilder consonantsBuilder = new StringBuilder();
        HashSet<Character> vowels = new HashSet<>();
        vowels.add('a');
        vowels.add('e');
        vowels.add('i');
        vowels.add('o');
        vowels.add('u');

        for (int i = 0; i < my_string.length(); i++) {
            if (!vowels.contains(my_string.charAt(i))) {
                consonantsBuilder.append(my_string.charAt(i));
            }
        }

        return consonantsBuilder.toString();
    }
}

// 모범 풀이
class Solution {
    public String solution(String my_string) {
        return my_string.replaceAll("[aeiou]", "");
    }
}